README.md

```diff
@@ -71,6 +71,34 @@ dq deploy /path/to/config.json
 
 Deploys this Data Quality repository to a remote.
 
+### Generate
+
+```
+dq generate GeneratorClass http://endpoint_to_data_sources /path/to/config/file
```

Comment:
@georgiana-b definitely do not want to expose `CkanGenerator` (or whatever the name of the class is) to the CLI. Please do it as requested in the issue: `ckan`, and internally have a mapping of keywords (like `ckan`) to classes (like `CkanGenerator`).